### PR TITLE
8271507: ProblemList SA tests that are failing with ZGC due to JDK-8248912

### PR DESCRIPTION
**Pull request by plummercj**
These tests recently started failing frequently with ZGC

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8271507](https://bugs.openjdk.java.net/browse/JDK-8271507): ProblemList SA tests that are failing with ZGC due to JDK-8248912

### Reviewers
 * [Daniel D. Daugherty](https://openjdk.java.net/census#dcubed) (@dcubed-ojdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk17 pull/294/head:pull/294` \
`$ git checkout pull/294`

Update a local copy of the PR: \
`$ git checkout pull/294` \
`$ git pull https://git.openjdk.java.net/jdk17 pull/294/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 294`

View PR using the GUI difftool: \
`$ git pr show -t 294`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk17/pull/294.diff">https://git.openjdk.java.net/jdk17/pull/294.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/ProblemList-zgc.txt [hunk 1]
```diff
@@ -41,6 +41,10 @@ serviceability/sa/ClhsdbPstack.java#id1                       8268722   macosx-x
 serviceability/sa/TestJmapCore.java                           8268722,8268283   macosx-x64,linux-aarch64
 serviceability/sa/TestJmapCoreMetaspace.java                  8268722,8268636   generic-all
 
+serviceability/sa/TestJhsdbJstackMixed.java                   8248912   generic-all
+serviceability/sa/ClhsdbPstack.java#id0                       8248912   generic-all
+serviceability/sa/ClhsdbPstack.java#id1                       8248912   generic-all
```
Comment (dcubed-ojdk) [inline test/hotspot/jtreg/ProblemList-zgc.txt:46]: I'm only seeing failures for 8248912 on linux-x64. You might
want to limit it to that platform.

**Review by dcubed-ojdk — APPROVED** [review]

File: test/hotspot/jtreg/ProblemList-zgc.txt (hunk 1 above)
Comment (plummercj) [reply test/hotspot/jtreg/ProblemList-zgc.txt:46]: I considered it, but given it's ZGC, and the SA support for it is not that stable, I thought it best just to silence it on all platforms.

Comment (dcubed-ojdk) [reply test/hotspot/jtreg/ProblemList-zgc.txt:46]: Sounds like a reasonable plan. Thumbs up.

**Review by dcubed-ojdk — APPROVED** [review]

**Comment by dcubed-ojdk** [comment]
Also, this is a trivial fix.

**Pull request closed without merging**